### DISCUSSION TITLE
We want to add the meeting list to homepage sidebar for today's meetings

### DISCUSSION THREAD

**Discussion by leendabolt** [code4recovery/12-step-meeting-list#1217] [General]
And then from there when users click "more information" or whatever is there that the "jump" will take them to a "today's meeting view?  How do we do that?

**Comment by joshreisner** [comment]
hi @leendabolt if you click "Customize" in WordPress do you see a list of Widgets you can add to the homepage sidebar? We have a widget called "Upcoming Meetings" that might work for that

here's what it looks like on our site:
![Screenshot 2023-09-18 at 1 15 29 PM](https://github.com/code4recovery/12-step-meeting-list/assets/1551689/151aafdc-8add-49c5-9be2-fce9bd6f7043)

**Reply by joshreisner** [reply]
@leendabolt sure so you get to this screen by logging into the WP backend and going to Appearance > Customize, or, once you are logged in, you should see a Customize link in the black "admin bar" at the top of the page

<img width="526" alt="Screenshot 2023-09-18 at 2 49 38 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/b91091b0-3ac1-453e-a785-e4321b70223d">

**Comment by leendabolt** [comment]
Hmm I am a novice - so - how did you get to that screen right there?
